src/LeanCode.ContractsGenerator/AnalyzerCodes.cs

```diff
@@ -0,0 +1,13 @@
+namespace LeanCode.ContractsGenerator.Analyzers;
+
+public static class AnalyzerCodes
+{
+    public const string InternalStructureIsWrong = "CNTR0001";
+    public const string UnsupportedKnownType = "CNTR0002";
+    public const string DuplicateErrorCodes = "CNTR0003";
+    public const string InternalTypeIsNotKnown = "CNTR0004";
+    public const string UnsupportedType = "CNTR0005";
+    public const string TopicDoesNotProduceNotification = "CNTR0006";
+    public const string TopicProducesNullableNotification = "CNTR0007";
+    public const string TopicMustProduceInternalType = "CNTR0008";
```

Comment:
Nitpick: `CNTR0008` doesn't follow the naming pattern used for other codes, as it's saying what should be rather than what's wrong.

Reply:
I would say there is no pattern altogether (e.g. `UnsupportedType` vs `TopicDoesNotProduceNotification`) and I wasn't really aiming for that.

Reply:
It just caught my attention that the last one is worded differently and stands out a bit, but yeah, you're right that it's an overstatement that there is a pattern. I think it would be nice if those would have a consistent style, but that's a nitpick, a really minor one.